internal/pkg/service/buffer/store/model/model.go

```diff
@@ -153,19 +153,34 @@ func ParseTableID(v string) (TableID, error) {
 
 type File struct {
 	key.FileKey
-	Mapping             Mapping          `json:"mapping" validate:"required,dive"`
-	StorageResource     *storageapi.File `json:"storageResource" validate:"required"`
-	ClosedAt            string           `json:"closedAt,omitempty"`
-	AllSlicesUploadedAt string           `json:"allSlicesUploadedAt,omitempty"`
-	ManifestUploadedAt  string           `json:"manifestUploadedAt,omitempty"`
-	ImportStartedAt     string           `json:"importStartedAt,omitempty"`
-	ImportFinishedAt    string           `json:"importFinishedAt,omitempty"`
+	Mapping            Mapping          `json:"mapping" validate:"required,dive"`
+	StorageResource    *storageapi.File `json:"storageResource" validate:"required"`
+	ClosingAt          *time.Time       `json:"closingAt,omitempty"`
+	ClosedAt           *time.Time       `json:"closedAt,omitempty"`
+	ManifestUploadedAt *time.Time       `json:"manifestUploadedAt,omitempty"`
+	ImportedAt         *time.Time       `json:"importedAt,omitempty"`
+	FailedAt           *time.Time       `json:"failedAt,omitempty"`
+	ImportStartedAt    *time.Time       `json:"importStartedAt,omitempty"`
+	ImportFinishedAt   *time.Time       `json:"importFinishedAt,omitempty"`
+	LastError          string           `json:"lastError,omitempty"`
```

Comment:
Upravil som trocha tieto fields, vsetky su `omittempty`, tak sa to zatial neprejavilo.